python/paddle/distributed/communication/broadcast.py

```diff
@@ -70,10 +80,12 @@ def broadcast(tensor, src, group=None, sync_op=True):
     )
 
 
-def broadcast_object_list(object_list, src, group=None):
+def broadcast_object_list(
+    object_list: list[dict[str, Tensor]], src: int, group: Group | None = None
```

Comment:
```suggestion
    object_list: list[object], src: int, group: Group | None = None
```

后面代码对这个输入没啥特殊要求，用 `object` 就行，用 `Any` 也可以，`Any` 的限制更少一些 ～

Reply:
收到